_search-plugins/collapse-search.md

```diff
@@ -0,0 +1,231 @@
+
+To collapse search results by the `item` field and sort them by `price`, you can use the following query:
+
+**Collapsed search results on item field**
+
+```json
+GET /bakery-items/_search
+{
+  "query": {
+    "match": {
+      "category": "cakes"
+    }
+  },
+  "collapse": {
+    "field": "item"
+  },
+  "sort": ["price"]
+}
+```
+
+**Response**
+
+```json
+{
+  "took": 3,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 4,
+      "relation": "eq"
+    },
+    "max_score": null,
+    "hits": [
+      {
+        "_index": "bakery-items",
+        "_id": "mISga5EB2HLDXHkv9kAr",
+        "_score": null,
+        "_source": {
+          "item": "Vanilla Cake",
+          "category": "cakes",
+          "price": 12,
+          "baked_date": "2023-07-02T00:00:00Z",
+          "baker": "Baker A"
+        },
+        "fields": {
+          "item": [
+            "Vanilla Cake"
+          ]
+        },
+        "sort": [
+          12
+        ]
+      },
+      {
+        "_index": "bakery-items",
+        "_id": "loSga5EB2HLDXHkv9kAr",
+        "_score": null,
+        "_source": {
+          "item": "Chocolate Cake",
+          "category": "cakes",
+          "price": 15,
+          "baked_date": "2023-07-01T00:00:00Z",
+          "baker": "Baker A"
+        },
+        "fields": {
+          "item": [
+            "Chocolate Cake"
+          ]
+        },
+        "sort": [
+          15
+        ]
+      }
+    ]
+  }
+}
+```
+
+The collapsed search results example will show only one "Chocolate Cake" entry, demonstrating how collapsing works in reducing redundancy.
+
+Collapsing affects only the top search results and does not change any aggregation results. The total number of hits shown in the response reflects all matching documents before collapsing is applied, including duplicates. However, the response doesn't tell you the exact number of unique groups formed by collapsing.
+
+---
+
+## Expanding collapsed results
+
+You can expand each collapsed top hit with the `inner_hits` property. 
+
+The following example request applies `inner_hits` to retrieve the lowest priced and most recent items for each type of cake.
+
+```json
+GET /bakery-items/_search
+{
+  "query": {
+    "match": {
+      "category": "cakes"
+    }
+  },
+  "collapse": {
+    "field": "item",
+    "inner_hits": [
+      {
+        "name": "cheapest_items",
+        "size": 1,
+        "sort": ["price"]
+      },
+      {
+        "name": "newest_items",
+        "size": 1,
+        "sort": [{ "baked_date": "desc" }]
+      }
+    ]
+  },
+  "sort": ["price"]
+}
+
+```
+
+### Multiple inner hits for each collapsed hit
+
+To obtain several groups of inner hits for each collapsed result, you can set different criteria for each group. For example, you could request the three lowest priced items and the three most recent items for every bakery item.
+
```

Comment:
Line above: Is "recent items for every bakery item" redundant?

Reply:
Addressed this and made it read clearer.